src/index.html

```diff
@@ -1,13 +1,373 @@
+              <div class="testimonial__position">Portable bluetooth</div>
+              <div class="testimonial__description">$ 129.00</div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section gallery" id="gallery">
+        <div class="container">
+          <h2 class="page__section--title">
+            Browse Bose
+            <br>
+            products by category
+          </h2>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-4 grid__item--1-8">
+                <img
+                  src="images/photos/gallery1.jpg"
+                  alt="location image"
+                  class="location__image location__image--1"
+                >
+              </div>
+              <div class=" location__image--container grid__item--tablet--5-6 grid__item--9-12">
+                <img
+                  src="images/photos/gallery2.jpg"
+                  alt="location image"
+                  class="location__image location__image--2"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Headphones & earbuds</h3>
+            </div>
+          </div>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-3 grid__item--1-6">
+                <img
+                  src="images/photos/gallery3.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+              <div class="location__image--container grid__item--tablet--4-6 grid__item--7-12">
+                <img
+                  src="images/photos/gallery4.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Speakers</h3>
+            </div>
+          </div>
+
+          <div class="location gallery__location">
+            <div class="location__images">
+              <div class="location__image--container grid__item--tablet--1-2 grid__item--1-4">
+                <img
+                  src="images/photos/gallery5.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+              <div class="location__image--container grid__item--tablet--3-6 grid__item--5-12">
+                <img
+                  src="images/photos/gallery6.jpg"
+                  alt="location image"
+                  class="location__image"
+                >
+              </div>
+            </div>
+            <div class="location__description">
+              <h3>Audio sunglasses</h3>
+            </div>
+          </div>
```

Comment:
font size is not right
and button is too much in width for tablet mode and for desktop its too wide
<img width="724" alt="image" src="https://user-images.githubusercontent.com/100297341/194122638-d590c371-2688-44ec-8a1c-cf8c1340ab82.png">
<img width="661" alt="image" src="https://user-images.githubusercontent.com/100297341/194122718-8d15df30-6109-4ac0-8dbd-cb4ea09e75f5.png">